src/main/java/com/indeed/jiraactions/api/IssuesAPICaller.java

```diff
@@ -132,17 +134,16 @@ private String getBasicInfoURL() throws UnsupportedEncodingException {
         return url;
     }
 
-    protected static final DateTimeZone JIRA_TIME_ZONE = DateTimeZone.forID("America/Chicago");
     private static final DateTimeFormatter JIRA_TIME_FORMAT = DateTimeFormat.forPattern("yyyy-MM-dd HH:mm");
     /**
      * Imhotep builders always runs in UTC-6, regardless of DST. However, Jira observes daylight savings time.
```

Comment:
This comment is now incorrect. I think there's a way to rephrase to say that it is not recommended that Imhotep builders observe DST.